services/core-api/app/api/document_generation/models/document_template.py

```diff
@@ -159,7 +165,7 @@ def delete_paragraph(paragraph):
                         break
 
         doc = None
-        if self.document_template_code in ('PMT', 'PMA', 'NPE', 'NCL', 'NWL', 'NRL'):
+        if self.document_template_code in ('PMT', 'PMA', 'NPE', 'NCL', 'NWL', 'NRL', 'ESP', 'ESL'):
```

Comment:
It could be a constant, but it would be a constant that is only used in this file.

Reply:
does it make sense for it to be a part of the data model?

Reply:
Since it's only used here I think it makes sense to keep it to the file so it requires less navigation to find the value. I'm up for putting it in the data table, but I don't think it's quite long enough "yet"

Reply:
Y'know what, I will adjust it so it's not even necessary 😄 It will just attempt to insert images into all templates.

It's too bad document insertion is not an open feature with Carbone and we had to implement it ourselves. We have to do the image insertion into the templates "dynamically" before sending the template document file to the document generator, which prevents us from using their caching feature (which is why this check was put in in the first place).